mobile/src/libs/components/button/button.tsx

```diff
@@ -8,12 +8,39 @@ import { styles } from './styles';
 type Properties = {
   label: string;
   onPress: () => void;
+  isDisabled?: boolean;
+  isSurvey?: boolean;
 };
 
-const Button: React.FC<Properties> = ({ label, onPress }) => {
+const Button: React.FC<Properties> = ({
+  label,
+  onPress,
+  isDisabled = false,
+  isSurvey = false,
```

Comment:
let's overwise provide custom className)

Reply:
@v0ldemar01 could you please explain what custom className here is? ive just created boolean, when there is survey, and button component already has styles for survey button so we just apply them when isSurvey is true

Reply:
We need to pass custom styling, we can use 2 approaches: using `className` property like frontend or pass boolean properties about button styling like colouring, outline mode etc. So let's proceed with second one, provide boolean property for changing color

Reply:
let's not pass custom className, but something like style: solid | outlined